package.json

```diff
@@ -85,6 +85,7 @@
   },
   "snyk": true,
   "engines": {
+    "node": ">=12.16.3 <13",
```

Comment:
The current ingest UI node is configured to be 14.16.0, so this fails [gitlab builds](https://gitlab.ebi.ac.uk/hca/ingest-ui/-/jobs/938351#L55)
```bash
error ingest-ui@0.2.10: The engine "node" is incompatible with this module. Expected version ">=12.16.3 <13". Got "14.16.0"
```